prime-router/src/main/kotlin/SubmissionReceiver.kt

```diff
@@ -108,6 +108,27 @@ abstract class SubmissionReceiver(
                 actionLogs.error(DuplicateSubmissionMessage(payloadName))
             }
         }
+
+        /**
+         * Determines what time of submission receiver to use based on [sender]
```

Comment:
For kdoc, in this program the preference is to use inline arguments, so next time try to describe the arguments in the description like:
Determines what type of submission receiver to use based on the [sender] based on a given [workflowEngine] ...